### PR TITLE
MC: Add protocol definitions for CP subsystem mgmt operations

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Core PR: https://github.com/hazelcast/hazelcast/pull/16226

### CODE REVIEW

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -741,3 +742,55 @@ methods:
...
+    since: 2.0
+    doc: |
+      Returns the current list of CP members.
+    request:
+      retryable: true
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: cpMembers
+          type: EntryList_String_String
```
Comment (sancar) [inline protocol-definitions/MC.yaml]: UUID has its own type in the protocol. We should not represent them as string.

Comment (puzpuzpuz) [reply protocol-definitions/MC.yaml]: Same applies to `Address`. But that means that I need to add at least one more custom data type, e.g. `EntryList_UUID_Address`. Does it sound OK to you?

Comment (sancar) [reply protocol-definitions/MC.yaml]: It is ok.It is better to have correct types.

Comment (puzpuzpuz) [reply protocol-definitions/MC.yaml]: Thanks. Going to fix it a bit later and ask you for another review round.

Comment (puzpuzpuz) [reply protocol-definitions/MC.yaml]: Done.

**Pull request merged**